_tsinfermodule.c

```diff
@@ -1274,8 +1274,9 @@ AncestorMatcher_init(AncestorMatcher *self, PyObject *args, PyObject *kwds)
     int ret = -1;
     int err;
     int extended_checks = 0;
+    int disable_weight_by_n = 0;
```

Comment:
Let's call it ``weight_by_n``, default=1 for this inferface. Agree with the way the C flags are done.